packages/p2p/src/components/orders/order-table/order-table-content.jsx

```diff
@@ -121,14 +109,25 @@ const OrderTableContent = ({ showDetails, is_active }) => {
                 </OrderTableHeader>
             );
         }
-
-        return <OrderEmptyStates />;
     }
 
-    return <OrderEmptyStates />;
+    return (
+        <div className='p2p-cashier__empty'>
+            <Icon icon='IcNoOrder' className='p2p-cashier__empty-icon' size={128} />
+            <div className='p2p-cashier__empty-title'>
+                <Localize i18n_default_text='You have no orders' />
+            </div>
+            {is_active && (
+                <Button primary large className='p2p-cashier__empty-button' onClick={() => changeTab(0)}>
+                    {localize('Buy/Sell')}
```

Comment:
```suggestion
                   <Localize i18n_default_text='Buy/Sell' />
```
Nitpick, just to keep it consistent with the above usage.